spec/classes/init_spec.rb

```diff
@@ -740,7 +740,7 @@
     end
 
     context 'as an array' do
-      let(:params) { { :package_name => %w(nscd foo) } }
+      let(:params) { { :package_name => %w[nscd foo] } }
```

Comment:
Please do turn off the cop that wants to change all the `%w()` to `%w[]`. This way we reduce code churn and make other PR's easier to merge. Funny thing is we went to `%w()` because of rubocop :>

Other than that, everything looks good! Thanks!